daemon/daemon_main.go

```diff
@@ -366,6 +366,9 @@ func init() {
 	flags.Bool(option.BlacklistConflictingRoutes, defaults.BlacklistConflictingRoutes, "Don't blacklist IP allocations conflicting with local non-cilium routes")
 	option.BindEnv(option.BlacklistConflictingRoutes)
 
+	flags.Bool(option.AnnotateK8sNode, defaults.AnnotateK8sNode, "Annotate Kubernetes nodes")
```

Comment:
Small nit, can you put this in line 359 just to be in alphabetical order?

Reply:
Done.